index.js

```diff
@@ -63,16 +62,17 @@ class EmergenceStress {
     this.#page = page
   }
 
-  static async withRandom () {
-    const name = randStr()
+  static async withRandom (lastChar) {
+    const name = randStr()+lastChar
     return await EmergenceStress.withName(name)
   }
 
   static async withName (name) {
     console.log(`loading fresh page`)
 
     const browser = await puppeteer.launch({
-      headless: 'new'
+      headless: 'new',
+      executablePath: 'chromium'
```

Comment:
this probably breaks on other systems. maybe an env var to set this would make sense. what's a good default?

Reply:
The default download is pretty small and cached. I'd say the default is a good default : ) - but sure, you can have an env to specify the executable path.

Reply:
the downwloaded binary didn't work for me. i just added a nix flake and added a env var to detect the executable dynamically. should be fine for now :blush: